src/functionalTest/resources/features/F-001/F-001_Test_Base_data.td.json

```diff
@@ -0,0 +1,97 @@
+{
+  "_guid_": "F-460_Test_Data_Base_Get_Dcoument",
+  "productName": "CCD Data Store",
+  "operationName": "Get Case for Case Worker",
+  "specs": [
+    "Get the document from Document Store"
+  ],
+  "method": "GET",
+  "uri": "/documents/{documentId}",
+  "user": {
+    "username": "befta.caseworker.2.solicitor.3@gmail.com",
+    "password": "Pa55word11"
+  },
+  "request": {
+    "headers": {
+      "_extends_": "Common_Request_Headers"
+    },
+    "pathVariables": {
+      "documentId": "fcdca3b6-607f-41af-ad8c-92267c921d44"
+    }
+  },
+  "expectedResponse": {
+    "responseCode" : 200,
+    "responseMessage" : "OK",
+    "headers": {
+      "Access-Control-Allow-Credentials" : "true",
+      "Access-Control-Allow-Headers" : "undefined",
+      "Access-Control-Allow-Methods" : "GET,POST,OPTIONS,PUT,DELETE",
+      "Cache-Control" : "[[ANY]]",
+      "connection" : "[[ANY]]",
+      "content-type" : "[[ANY]]",
+      "date" : "[[ANY]]",
+      "Expires" : "0",
+      "Pragma" : "no-cache",
+      "request-context" : "[[ANY]]",
+      "Strict-Transport-Security" : "[[ANY]]",
```

Comment:
placeholder value [[ANY]] is no more available. See available values here:
https://github.com/hmcts/befta-fw/blob/master/src/main/java/uk/gov/hmcts/befta/util/ExpectedValuePlaceholder.java